packages/block-serialization-default-parser/parser.php

```diff
@@ -62,6 +67,19 @@ class WP_Block_Parser_Block {
 	 */
 	public $innerContent;
 
+	/**
+	 * Constructor.
+	 *
+	 * Will populate object properties from the provided arguments.
+	 *
+	 * @since 3.8.0
```

Comment:
> If everything in the Gutenberg 4.8 milestone will ship in WP 5.1 then all @since PHPDoc blocks should be @since 5.1.0

The issue here is that, we can't know for sure when merging the PR. 5.0.3 is still undetermined for the moment.

Reply:
It wouldn't be a bad idea to keep `@since x.y.z` in the code and only update with a tool when preparing the next release for WordPress. Would `phpcs` be fine with that? :)

I would also always use `@package WordPress` and i18n functions without 2nd param for PHP files located in `packages` folder.

Let's agree on something and possibly enforce as much as possible with `phpcs`.